precompiles/dapps-staking/DappsStaking.sol

```diff
@@ -8,46 +8,54 @@ interface DappsStaking {
 
     // Storage getters
 
-    /// @dev Read current era.
+    /// @notice Read current era.
     /// @return The current era
     function read_current_era() external view returns (uint256);
 
-    /// @dev Read unbonding period constant.
+    /// @notice Read unbonding period constant.
     /// @return The unbonding period in eras
     function read_unbonding_period() external view returns (uint256);
 
-    /// @dev Read Total network reward for the given era
+    /// @notice Read Total network reward for the given era
     /// @return Total network reward for the given era
     function read_era_reward(uint32 era) external view returns (uint128);
 
-    /// @dev Read Total staked amount for the given era
+    /// @notice Read Total staked amount for the given era
     /// @return Total staked amount for the given era
     function read_era_staked(uint32 era) external view returns (uint128);
 
-    /// @dev Read Staked amount for the staker
+    /// @notice Read Staked amount for the staker
     /// @return Staked amount for the staker
     function read_staked_amount(address staker) external view returns (uint128);
 
-    /// @dev Read the staked amount from the era when the amount was last staked/unstaked
+    /// @notice Read the staked amount from the era when the amount was last staked/unstaked
     /// @return The most recent total staked amount on contract
     function read_contract_stake(address contract_id) external view returns (uint128);
 
 
     // Extrinsic calls
 
-    /// @dev Register provided contract.
+    /// @notice Register provided contract.
     function register(address) external;
 
-    /// @dev Stake provided amount on the contract.
+    /// @notice Stake provided amount on the contract.
     function bond_and_stake(address, uint128) external;
 
-    /// @dev Start unbonding process and unstake balance from the contract.
+    /// @notice Start unbonding process and unstake balance from the contract.
     function unbond_and_unstake(address, uint128) external;
 
-    /// @dev Withdraw all funds that have completed the unbonding process.
+    /// @notice Withdraw all funds that have completed the unbonding process.
     function withdraw_unbonded() external;
 
-    /// @dev Claim contract's rewards.
-    /// @dev Please use only for testing. This call will be replaced with 2 new calls in PR#6
+    /// @notice Claim contract's rewards for the specified era.
+    ///
+    /// @dev This call is deprecated and shouldn't be used anymore.
     function claim(address, uint128) external;
```

Comment:
there are no projects in production yet. I'd remove this

Reply:
Great, that makes it much cleaner.